radixdlt-java/src/main/java/com/radixdlt/client/core/crypto/ECSignature.java

```diff
@@ -25,6 +29,21 @@ public ECSignature(BigInteger r, BigInteger s) {
 		this.s = Bytes.trimLeadingZeros(s.toByteArray());
 	}
 
+	public ECSignature(byte[] bytes) {
+		DLSequence seq;
+		ASN1Integer r, s;
+		try (ASN1InputStream decoder = new ASN1InputStream(bytes)) {
```

Comment:
Don't really have a problem with *how* this works, but would like to see this in a more explicit function instead. So something like ECSignature.fromASN1(bytes) or whatever it's called, which then internally calls to ECSignature(r, s)